code/game/gamemodes/cult/cult_items.dm

```diff
@@ -116,19 +116,215 @@
+	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade)
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	armor = list(melee = -100, bullet = -100, laser = -100,energy = -100, bomb = -100, bio = -100, rad = -100)
+	slowdown = -1
+	hooded = 1
+	hoodtype = /obj/item/clothing/head/berserkerhood
+	
+/obj/item/clothing/head/berserkerhood
+	name = "flagellant's robes"
+	desc = "Blood-soaked garb infused with dark magic; allows the user to move at inhuman speeds, but at the cost of increased damage."
+	icon_state = "culthood"
+	body_parts_covered = HEAD
+	flags = NODROP
+	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
+	armor = list(melee = -100, bullet = -100, laser = -100,energy = -100, bomb = -100, bio = -100, rad = -100)
+	
+/obj/item/clothing/suit/hooded/cultrobes/berserker/pickup(mob/living/user)
+	..()
+	if(!iscultist(user))
+		user << "<span class='cultlarge'>\"I wouldn't advise that.\"</span>"
+		user << "<span class='warning'>An overwhelming sense of nausea overpowers you!</span>"
+		user.Dizzy(30)
+		user.Weaken(5)
+   	    	
+/obj/item/clothing/glasses/night/cultblind
+	desc = "May nar-sie guide you through the darkness and shield you from the light."
+	name = "zealot's blindfold"
+	icon_state = "blindfold"
+	item_state = "blindfold"
+	darkness_view = 8
+	flash_protect = 1
+	
+/obj/item/clothing/glasses/night/cultblind/pickup(mob/living/user)
+	..()
+	if(!iscultist(user))
+		user << "<span class='cultlarge'>\"I wouldn't advise that.\"</span>"
+		user << "<span class='warning'>An overwhelming sense of nausea overpowers you!</span>"
+		user.Dizzy(30)
+		user.Weaken(5)
+
+/obj/item/weapon/reagent_containers/food/drinks/bottle/unholywater
+	name = "flask of unholy water"
+	desc = "Toxic to nonbelievers; this water renews and reinvigorates the faithful of nar'sie."
+	icon_state = "holyflask"
+	color = "#333333"
+	list_reagents = list("unholywater" = 40)
+
+/obj/item/device/shuttle_curse
+	name = "cursed orb"
+	desc = "You peer within this smokey orb and glimpse terrible fates befalling the escape shuttle."
+	icon = 'icons/obj/projectiles.dmi'
+	icon_state ="bluespace"
+	color = "#ff0000"
+	var/global/curselimit = 0
+	
+/obj/item/device/shuttle_curse/attack_self(mob/user)
+	if(!iscultist(user))
+		user.Weaken(5)
+		user << "<span class='warning'>A powerful force shoves you away from [src]!</span>"
+		return
+	if(curselimit > 1)
+		user << "<span class='notice'>We have exhausted our ability to curse the shuttle.</span>"
+		return
+	if(SSshuttle.emergency.mode == SHUTTLE_CALL)
+		var/cursetime = 1500
+		var/timer = SSshuttle.emergency.timeLeft(1) + cursetime
+		SSshuttle.emergency.setTimer(timer)
+		user << "<span class='danger'>You shatter the orb! A dark essence spirals into the air, then disappears.</span>"
+		playsound(user.loc, "sound/effects/Glassbr1.ogg", 50, 1)
+		qdel(src)
+		sleep(20)
+		var/global/list/curses = list("A fuel technician just slit his own throat and begged for death. The shuttle will be delayed by two minutes.",
```

Comment:
I didn't see a repeat after the change but I didn't test that hard for it.

Where could I move the global definition so that making a new _______ wouldn't redefine it?